scripts/capture_and_parse_imu_serial_data.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+
+# Set the default serial port, baud rate, and capture duration (in seconds)
+DEFAULT_SERIAL_PORT="/dev/cu.usbmodem2101"
```

Comment:
Maybe make OS agnostic? For linux this is /dev/ttyACM0. Could perform check in script.

Reply:
Not a huge worry

Reply:
I gotchu that way we can just use the defaults